crates/astria-conductor/src/executor.rs

```diff
@@ -209,7 +203,7 @@ impl<C: ExecutionClient> Executor<C> {
         // get transactions for our namespace
         let Some(txs) = block.rollup_txs.get(&self.namespace) else {
```

Comment:
Suggestion to avoid the allocation/collection further down (see following commoents for the rest):
```suggestion
        let Some(txs) = block.rollup_txs.remove(&self.namespace) else {
```